lib/json_api_controller.rb

```diff
@@ -56,6 +56,22 @@ def serializer
     @serializer ||= "#{ resource_name.camelize }Serializer".constantize
   end
 
+  def policy_object
```

Comment:
If you `include Pundit` you get a `policy` method, which is a better name, but I didn't want to confuse us with module inclusion order for now. I'll rename it at a later PR.